lib/sherlock_homes/driver.rb

```diff
@@ -0,0 +1,56 @@
+module SherlockHomes
+  class Driver
+
```

Comment:
Another future PR:  Can we make the driver not be global?  When we get to workflows, we might want to use different drivers for different sites.  (I saw there was a capybara driver that just used RestClient or something like that behind the scenes). Might be nice to use that for non-js sites. (Created a pivotal story in the icebox for this)